theme.json

```diff
@@ -1105,6 +1105,9 @@
 						}
 					}
 				}
+			},
+			"core/list": {
+				"css": "& {list-style-type:disc;} & li{margin-bottom: 0.5rem;}"
 			}
```

Comment:
The list style type needs to be removed because it overrides the user's block setting.
Users need to be able to set the list to _ordered_.